parlai/tasks/eli5/build.py

```diff
@@ -0,0 +1,113 @@
+        'e3a8addc6a33b54b1dd6488a98c875851ef1aca3b80133d39f6897330a8835fb',
+        zipped=False,
+    ),
+    DownloadableFile(
+        'https://dl.fbaipublicfiles.com/eli5qa/explainlikeimfive_ccrawl_ids.json.gz',
+        'explainlikeimfive_ccrawl_ids.json.gz',
+        '59cd7b6a8580421aecae66cd33d065073f2abf21d86097b3262bd460a7a14f0d',
+        zipped=False,
+    ),
+    DownloadableFile(
+        'https://dl.fbaipublicfiles.com/eli5qa/explainlikeimfive_unigram_counts.json',
+        'explainlikeimfive_unigram_counts.json',
+        '0433a4dda7532ba1dae2f5b6bf70cd5ab91fd2772f75e99b4c15c2e04ba80dfd',
+        zipped=False,
+    ),
+]
+
+
+# Setup a directory called dir_name at dpath
+def setup_dir(dpath, dir_name):
+    if not isdir(pjoin(dpath, dir_name)):
+        subprocess.run(['mkdir', pjoin(dpath, dir_name)])
+
+
+def build(opt):
+    dpath = pjoin(opt['datapath'], 'eli5')
+    pre_computed_path = pjoin(dpath, 'pre_computed')
+    processed_path = pjoin(dpath, 'processed_data')
+
+    version = '1.0'
+
+    if not build_data.built(dpath, version_string=version):
+        print('[building image data: ' + dpath + ']')
+
+        # Setup directory folders
+        setup_dir(dpath, 'tmp')
+        setup_dir(dpath, 'pre_computed')
+        setup_dir(dpath, 'processed_data')
+        setup_dir(processed_path, 'collected_docs')
+
+        if build_data.built(dpath):
+            # An older version exists, so remove these outdated files.
+            build_data.remove_dir(dpath)
+        build_data.make_dir(dpath)
+
+        # Download the pre-computed data.
+        for downloadable_file in RESOURCES:
+            downloadable_file.download_file(pre_computed_path)
+            if downloadable_file.file_name.endswith('.gz'):
+                # Check file hasn't already been unzipped
+                if not isfile(
+                    pjoin(pre_computed_path, downloadable_file.file_name[:-3])
+                ):
+                    print(f'Unzipping {downloadable_file.file_name}')
+                    subprocess.run(
+                        [
+                            'gunzip',
+                            pjoin(pre_computed_path, downloadable_file.file_name),
+                        ],
+                        stdout=subprocess.PIPE,
+                    )
+
+        # Check that wet.paths and eli5 ccrawl ids have been unzipped
+        if not isfile(pjoin(pre_computed_path, 'wet.paths')):
+            print(
+                f'Unzip wet.paths.gz located at {pre_computed_path} with command:\n',
+                'gunzip wet.paths.gz',
+            )
+        if not isfile(pjoin(pre_computed_path, 'explainlikeimfive_ccrawl_ids.json')):
+            print(
+                f'Unzip explainlikeimfive_ccrawl_ids.json.gz located at {pre_computed_path} with command:\n',
+                'gunzip explainlikeimfive_ccrawl_ids.json.gz',
+            )
+        # Mark the data as built.
+        build_data.mark_done(dpath, version_string=version)
```

Comment:
do we want to mark as `built` even if the files are not unzipped accordingly? we don't need to redownload each call to `build` but maybe we should print these instructions each time `build.py` is called?